tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -1299,6 +1299,36 @@ def test_generate_messages_with_defaults():
             "x-forwarded-for": "00.000.00.000, 00.000.000.000",
         },
     ],
+    "profiler.result.get_expectation_suite": [
+        {
+            "event": "profiler.result.get_expectation_suite",
+            "event_payload": {
+                "anonymized_expectation_suite_name": "4b6bf73298fcc2db6da929a8f18173f7"
+            },
+            "success": True,
+            "version": "1.0.0",
+            "event_time": "2020-06-25T16:08:23.570Z",
+            "data_context_id": "00000000-0000-0000-0000-000000000002",
+            "data_context_instance_id": "10000000-0000-0000-0000-000000000002",
+            "ge_version": "0.11.9.manual_testing",
+            "x-forwarded-for": "00.000.00.000, 00.000.000.000",
+        },
+    ],
+    "data_assistant.result.get_expectation_suite": [
+        {
+            "event": "data_assistant.result.get_expectation_suite",
+            "event_payload": {
```

Comment:
just want to make sure this is representative of _all_ possible payload values. it's not required in order to infer the schema, but for my sake i think i'd like to have the examples be as such.

Reply:
@Super-Tanner Confirmed -- yes, and the new test messages verify that.  Thanks.